Composer/packages/types/src/server.ts

```diff
@@ -38,16 +38,54 @@ export type ISettingManager = {
   getFileName: () => string;
 };
 
-export type DirectLineLogType = 'Error' | 'Warn' | 'Info';
-
 export interface DirectLineError {
   status: number;
   message: string;
   details?: string;
 }
 
-export interface DirectLineLog extends DirectLineError {
+/** Types of Web Chat traffic that can be sent to the client */
+export type ConversationTrafficItem =
+  | ConversationActivityTrafficItem
+  | ConversationNetworkTrafficItem
+  | ConversationNetworkErrorItem;
+
+export type NetworkTrafficRequest = {
+  method: string;
+  payload: any;
+  route: string;
+};
+
+export type NetworkTrafficResponse = {
+  payload: any;
+  statusCode: number;
+};
+
+export type ConversationNetworkTrafficItem = {
+  request: NetworkTrafficRequest;
+  response: NetworkTrafficResponse;
   timestamp: string;
-  route?: string;
-  logType: DirectLineLogType;
-}
+  trafficType: 'network';
+};
+
+export type ConversationActivityTraffic = {
+  activities: any[];
```

Comment:
We have botframework-types package. Can we include Activity type from there?

Reply:
Good call, let me try that